### DISCUSSION TITLE
Is it possible to run the official vscode-cli

### DISCUSSION THREAD

**Discussion by 54145a** [termux/termux-app#4071] [Q&A] [answered]
See https://code.visualstudio.com/download

**Comment by TomJo2000** [comment, marked as answer]
VSCode and Code Server are available from the TUR.
- https://github.com/termux-user-repository/tur/tree/master/tur/code-oss
- https://github.com/termux-user-repository/tur/tree/master/tur/code-server

**Reply by 54145a** [reply]
Thankyou!
